src/Exception/ResetPasswordExceptionInterface.php

```diff
@@ -14,5 +14,8 @@
  */
 interface ResetPasswordExceptionInterface extends \Throwable
 {
+    public const MESSAGE_PROBLEM_VALIDATE = 'There was a problem validating your password reset request';
+    public const MESSAGE_PROBLEM_HANDLE = 'There was a problem handling your password reset request';
```

Comment:
Those strings are hardcode in Maker - probably we could migrate them in this bundle as well and then just reuse these constants in Maker, see https://github.com/symfony/maker-bundle/blob/085358c107a7047b4889455d7518d0dbf55169f1/src/Resources/skeleton/resetPassword/ResetPasswordController.tpl.php#L160